src/Compute/Compute/Generated/VirtualMachineScaleSet/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -37,7 +37,9 @@
 
 namespace Microsoft.Azure.Commands.Compute.Automation
 {
-    [GenericBreakingChange("Starting in May 2023 the \"New-AzVmss\" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch")]
+    [GenericBreakingChange("Starting May 2023, the \"New-AzVmss\" cmdlet will deploy with the Trusted Launch configuration by default. To know more about Trusted Launch, please visit https://docs.microsoft.com/en-us/azure/virtual-machines/trusted-launch", "6.0.0")]
+    [GenericBreakingChange("Starting May 2023, the \"New-AzVmss\" cmdlet will create a OrchestrationMode: Flexible VMSS by default unless explicitly provded \"-OrchestrationMode Uniform\".", "6.0.0")]
+    [GenericBreakingChange("Starting May 2023, the \"New-AzVmss\" used with \"-OrchestrationMode Flexible\" will allow NAT pool to be configured. Before, VMSS in OrchestrationMode:Flexible defaulted NAT pool to be not configured(null).", "6.0.0")]
```

Comment:
should the version I wrote: 6.0.0, be Az.Compute version or Az version? 
I wrote 6.0.0 because with breaking change we will be increasing major version, right?

Reply:
Correct. Please add change log